nodes/move-object/move-object.js

```diff
@@ -0,0 +1,153 @@
+module.exports = function (RED) {
+  "use strict";
+  const { S3 } = require("@aws-sdk/client-s3");
+
+  // Move Object
+  function S3MoveObject(n) {
+    RED.nodes.createNode(this, n); // Getting options for the current node
+    this.conf = RED.nodes.getNode(n.conf); // Getting configuration
+    var node = this; // Referencing the current node
+    var config = this.conf ? this.conf : null; // Cheking if the conf is valid
+
+    // If there is no conifg
+    if (!config) {
+      node.warn(RED._("Missing S3 Client Configuration!"));
+      return;
+    }
+
+    this.on("input", async function (msg, send, done) {
+      let bucket = n.bucket !== "" ? n.bucket : null; // Bucket info
+      let key = n.key !== "" ? n.key : null; // Object key
+
+      let sourcekey = n.sourcekey !== "" ? n.sourcekey : null;
```

Comment:
The property names in the ***.js*** file should correspond with the property names in the ***.html*** file for the node, in this instance `n.sourcekey` should be changed to `n.oldobject`. Please refer to the comments in the ***.html*** file